elasticsearch_dsl/_async/index.py

```diff
@@ -109,6 +150,19 @@ def as_template(
             template_name, pattern or self._name, index=self, order=order
         )
 
+    def as_composable_template(
+        self,
+        template_name: str,
+        pattern: Optional[str] = None,
+        priority: Optional[int] = None,
+    ) -> AsyncComposableIndexTemplate:
+        # TODO: should we allow pattern to be a top-level arg?
+        # or maybe have an IndexPattern that allows for it and have
+        # Document._index be that?
```

Comment:
It seems to me that we should either do it or remove this comment. (The current code seems perfectly usable to me.)

Reply:
Okay, I'll remove, since these are likely from very long ago and we clearly are fine with the way this is.